main.js

```diff
@@ -74,7 +74,9 @@ if (process.env.NODE_ENV === 'production') {
 	module.exports.errorHandler = function () {
 		const middleware = _errorHandler();
 		return (err, req, res, next) => {
-			logger.error(err, { event: 'uncaughterror' });
+			if (!res.locals.suppressRavenLogger) {
```

Comment:
**thought**: this isn't a variable that needs to be accessed by the templates, so i don't think `res.locals` makes sense? but also i wouldn't want us to just jam it straight on `res`. mainly i just don't like how express middleware makes it difficult to understand where stuff is actually coming from. idk

Reply:
Yeah I thought about adding it to `request` but that felt wrong somehow as well 😔